cli/util/util.go

```diff
@@ -899,3 +899,24 @@ func InstantiateFileFromTemplate(templatePath string, templateContent string,
 	}
 	return nil
 }
+
+// CollectAppList collects all the supposed applications from the passed directory.
+func CollectAppList(baseDir string) ([]string, error) {
+	dirEnrties, err := os.ReadDir(baseDir)
+	if err != nil {
+		return nil, err
+	}
+
+	apps := make([]string, 0)
+	for _, entry := range dirEnrties {
+		dirItem := filepath.Join(baseDir, entry.Name())
+		if IsApp(dirItem) {
+			apps = append(apps, entry.Name())
+		} else {
+			log.Warnf("The application %s can't be indexed: failed to access the source",
```

Comment:
Can we just write that tt is skipping entry.Name(), because it is not an application? I don't understand what "failed to access the source" mean and what "indexing" we are talking about.

Reply:
Fixed, thank you.